app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def nice_datetime(date)
+    date.strftime('%c')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.